miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniAppDisplay.kt

```diff
@@ -29,4 +30,18 @@ interface MiniAppDisplay : LifecycleObserver {
      * within the same state of parent's lifecycle.
      */
     fun destroyView()
+
+    /**
+     * Provided the interface implementation for javascript interaction.
+     * @property miniAppMessageInterface The interface implementation which transmits data
+     * between native and miniapp.
+     */
+    fun injectJSInterface(miniAppMessageInterface: MiniAppMessageInterface)
```

Comment:
I'm not sure about this because this doesn't give us a guarantee that the native App has defined the interface. What will happen if the native app doesn't define the interface? I think the Mini App will have errors, right?